tests/tools/oesign/test-enclave/enclave/CMakeLists.txt

```diff
@@ -3,6 +3,16 @@
 
 set(EDL_FILE ../oesign_test.edl)
 
+if (DEFAULT_TEST_ENCLAVE_CRYPTO_LIB STREQUAL "MbedTLS")
```

Comment:
is this ok being case sensitive or is a TOLOWER operation needed here like I saw in another file in this PR?

Reply:
Thanks for the catch. Updated.